### PR TITLE
Bugfix After Selecting File From Search It Doesn't Open

### PR DESCRIPTION
**Pull request by alperozturk96**
Before:
https://github.com/nextcloud/android/assets/67455295/8218891c-880d-40c4-a92c-64feb76f00b0

After:
https://github.com/nextcloud/android/assets/67455295/08b00a98-f6e2-4990-a1e4-542f7441eb26

- [x] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 1]
```diff
@@ -174,7 +170,6 @@ public class FileDisplayActivity extends FileActivity
     public static final String RESTART = "RESTART";
     public static final String ALL_FILES = "ALL_FILES";
     public static final String LIST_GROUPFOLDERS = "LIST_GROUPFOLDERS";
-    public static final String PHOTO_SEARCH = "PHOTO_SEARCH";
```
Comment (alperozturk96) [inline app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java:177]: unused

File: app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 2]
```diff
@@ -207,7 +202,6 @@ public class FileDisplayActivity extends FileActivity
     public static final int REQUEST_CODE__MOVE_FILES = REQUEST_CODE__LAST_SHARED + 3;
     public static final int REQUEST_CODE__COPY_FILES = REQUEST_CODE__LAST_SHARED + 4;
     public static final int REQUEST_CODE__UPLOAD_FROM_CAMERA = REQUEST_CODE__LAST_SHARED + 5;
-    public static final int REQUEST_CODE__UPLOAD_SCAN_DOC_FROM_CAMERA = REQUEST_CODE__LAST_SHARED + 6;
```
Comment (alperozturk96) [inline app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java:210]: unused

**Comment by alperozturk96** [comment]
@ZetaTom Good point. When I search I expect to navigate from filtered files to file detail page. However in master and also in this branch we don't have trailing icons and long press functionality therefore after search user can only tap item and navigate to detail page. 

Why we are not adding trailing icons for filtered files as well? @tobiasKaminsky Could be new feature to implement.

Screenshot 2023-11-01 at 15 58 57

**Comment by alperozturk96** [comment]
@tobiasKaminsky New [issue](https://github.com/nextcloud/android/issues/12144) created for this improvement. This bugfix should be completed after implement the new feature in this regard.

**Comment by alperozturk96** [comment]
After merging this [PR](https://github.com/nextcloud/android/pull/12209#issuecomment-1831669674) we can review this PR.

**Review by JonasMayerDev — APPROVED** [review]
Works for me

**Pull request merged**